packages/ui-img/src/Img/__tests__/Img.test.js

```diff
@@ -60,106 +58,6 @@ describe('<Img />', () => {
 
   it('should render an overlay color', async () => {
     await mount(<Img src={image} overlay={{ color: '#ff0000', opacity: 7 }} />)
-    expect(await ImgLocator.find(`.${styles.overlay}`)).to.exist()
-  })
-
-  it('should render a blur filter', async () => {
```

Comment:
Why have these been deleted? Not possible to do with emotion?

Reply:
It is possible, though not necessary. We have to find a way to unit test the styles.js. These  tests will be covered there. (I'll create a ticket for that)